mlflow/entities/_mlflow_object.py

```diff
@@ -44,12 +43,6 @@ def __init__(self):
     def to_string(self, obj):
         if isinstance(obj, _MLflowObject):
             return "<%s: %s>" % (get_classname(obj), self._entity_to_string(obj))
-        # Handle nested lists inside MLflow entities (e.g. lists of metrics/params)
```

Comment:
Removed this code path as we no longer have lists nested inside of MLflow entities (we have dicts instead). Note that this means we don't truncate the string representations of runs with many distinct metrics/params/tags. I could try to add back such truncation logic, but it's not obvious to me that it'd be super useful.